README.md

```diff
@@ -15,86 +20,78 @@ A quick and small python script that helps you autotype on websites that have co
 <img src="https://i.imgur.com/pUfYwD0.gif">
 
 # Pre-requisites
+<hr>
+
 `python3.9.x`
 `pip`
 
-# Development Setup
-- Create a new virtual environment using `pipenv`
-```bash
-pip install pipenv --user
+# Installation
+<hr>
 
-# create venv and install dependencies from Pipfile
-pipenv install
-```
-- Activate the environment
+Install Autotype with pip
 ```bash
-pipenv shell
+  pip3 install Autotype
 
-# check if activated
-pip -V
-
-pip install -r requirements.txt
+  Autotype --help
 ```
 
-- For Linux
+Install from source
+
 ```bash
+git clone https://github.com/tushar5526/Autotype
 
-For Ubuntu or other distros with Apt:
+cd Autotype
 
-sudo apt-get install python3-tk
+python3 -m pip install .
 
-For Fedora:
+# To check if Autotype is correctly installed
+Autotype version
 
-sudo dnf install python3-tkinter
+```
 
-For Arch Based
+# Usage
+<hr>
 
-sudo pacman -S tk
+For Help and Listing all the Features of Autotype
+```bash
+
+Autotype type --help
 
 ```
 
-- Run it as CLI app
+Simulation of a Code file and Writing it.
+```bash
 
-> Provide the path of the file to be autotyped and the delay time through teminal/shell.
+Autotype type --path <path of the file to write> --delay <delay in seconds befor writing>
 
-```bash
-python3 command_line_script --path filePath --delay delay_before_typing
 ```
 
+<b> <i> 🗒️ Note: </i>  --path [Required] --delay[Optional [ default= 3seconds ] ]️</b>
 
-### Run the GUI if you are not familiar with CLI apps.
+<hr>
 
-- Run the script `python3 GUI_script.py`
-<img src="https://i.imgur.com/QhDjIqe.jpeg">
+# Dev-Setup
+```bash
+git clone https://github.com/tushar5526/Autotype
 
-- You can simply type your code in the textbox , enter the time delay and click the `Start Typing` button.The script will then type your code for you.
-<img src="https://i.imgur.com/3ysBzIT.gif">
+python3 -m venv venv
 
-- If your code is in a file , then leave the textbox blank , enter the time delay and click `Start Typing` button.A file exploror prompt will open asking you to select the file.Simply select your file and Done! Autotype will type your code for you.
-<img src="https://imgur.com/SOauxRx.gif">
+source venv/bin/activate
 
-- There are two themes in the GUI Script: Dark and Light. By clicking the toggle in the bottom left corner of the window, you can switch between the two.
-<img src="https://imgur.com/NjLfWcL.gif)">
+pip3 install -r requirements-dev.txt
```

Comment:
There is no requirements-dev.txt or requirements.txt. I see we are also not using pipenv and poetry for package management. What tool are we using here ?